src/components/dropdown/theme.css

```diff
@@ -6,7 +6,7 @@
 }
 
 .hx-dropdown-up {
-  margin-bottom: var(--dropdown-spacing);
+  margin-bottom: calc(-1 * var(--dropdown-spacing));
```

Comment:
I believe this needs `margin-top` instead as `margin-bottom` has no effect when using `top: XXXpx`

Reply:
Yes, margin-top is what I've tested locally. Creating commit via GitHub web GUI wasn't the best idea :)